README.md

```diff
@@ -49,8 +49,9 @@ SystemML computations can be executed in a variety of different modes. To begin
 can be operated in Standalone mode on a single machine, allowing data scientists to develop
 algorithms locally without need of a distributed cluster. In order to scale up, algorithms can also be distributed
 across Spark or Hadoop.
-This flexibility allows the utilization of an organization's existing resources and expertise. In addition, SystemML
-can be operated via Scala, Java, and Python. SystemML also features an embedded API for scoring models.
+This flexibility allows the utilization of an organization's existing resources and expertise.
+In addition, SystemML features a Spark MLContext API that allows for programmatic interaction via Scala and Java.
```

Comment:
Supposedly we also can interact with MLContext via Python using [SystemML.py](https://github.com/apache/incubator-systemml/blob/master/src/main/java/org/apache/sysml/api/python/SystemML.py).